ci-automation/vendor-testing/azure.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+# Copyright (c) 2021 The Flatcar Maintainers.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+set -x
+set -o noglob
+
+set -euo pipefail
+
+# Test execution script for the qemu vendor image.
+# This script is supposed to run in the SDK container.
+
+work_dir="$1"; shift
+arch="$1"; shift
+vernum="$1"; shift
+tapfile="$1"; shift
+
+# $@ now contains tests / test patterns to run
+
+source ci-automation/ci_automation_common.sh
+
+mkdir -p "${work_dir}"
+cd "${work_dir}"
+
+testscript="$(basename "$0")"
+
+# Fetch the Azure image if not present
+if [ -f "${AZURE_IMAGE_NAME}" ] ; then
+    echo "++++ ${testscript}: Using existing ${work_dir}/${AZURE_IMAGE_NAME} for testing ${vernum} (${arch}) ++++"
+else
+    echo "++++ ${testscript}: downloading ${AZURE_IMAGE_NAME} for ${vernum} (${arch}) ++++"
+    copy_from_buildcache "images/${arch}/${vernum}/${AZURE_IMAGE_NAME}.bz2" .
+    bunzip2 "${work_dir}/${AZURE_IMAGE_NAME}.bz2"
```

Comment:
```suggestion
    lbunzip2 "${work_dir}/${AZURE_IMAGE_NAME}.bz2"
```
We also use it in `qemu_update.sh`, I think it's ok to require it as dependency